web-wallet/src/lib/dusk/test-helpers/mockReadableStore.js

```diff
@@ -1,18 +1,13 @@
-import { get, writable } from "svelte/store";
+import { get, readable } from "svelte/store";
 
 /** @param {*} initialValue */
 function mockReadableStore (initialValue) {
-	const store = writable(initialValue);
-	const { set, subscribe } = store;
+	const store = readable(initialValue);
+	const { subscribe } = store;
 	const getMockedStoreValue = () => get(store);
 
-	/** @param {*} value */
-	const setMockedStoreValue = value => set(value);
-
 	return {
 		getMockedStoreValue,
-		set,
-		setMockedStoreValue,
```

Comment:
The point here was being able to inject values at will for tests in a store that is otherwise read only: that's why `setMockedStoreValue` was in place.